src/webclients/components/transformations/transformations.ts

```diff
@@ -236,10 +230,9 @@ export class PassPipeline extends TransformationListItem {
             'title': 'Run this pass with default parameters',
             'click': (event: Event) => {
                 event.stopPropagation();
-                vscode.postMessage({
-                    type: 'sdfv.apply_transformations',
-                    transformations: [this.json],
-                });
+                TransofrmationListPanel.getInstance().invoke(
+                    'applyTransformations', [[this.json]]
```

Comment:
shouldn't these be in enums? what do you think?

Reply:
Ideally they would be part of the autocompletion, not just enums. The issue is that decorators in TS (currently, at least) don't have any design-time capabilities, meaning that the linter has no idea about the registered ICPCRequests. I am hoping this becomes available at some point, because that would further, _drastically_ simplify the ICPC system.

Using enums until then is an alternative, but it requires additional changes for each new ICPCRequest written - so I am not a huge fan of it in this instance. However, there's a system that clearly logs any attempted call to a procedure that doesn't exist (i.e., typo detection), so bugs are easy to detect and fix.